src/libs/conduit/python/conduit_python.cpp

```diff
@@ -100,7 +100,11 @@ PyString_AsString(PyObject *py_obj)
                                                           "strict"); // Owned reference
         if(temp_bytes != NULL)
         {
+            #ifdef Py_LIMITED_API
+            res = _conduit_strdup(PyBytes_AsString(temp_bytes));
+            #else
             res = _conduit_strdup(PyBytes_AS_STRING(temp_bytes));
+            #endif
```

Comment:
It's probably better to just always use the ABI-stable code here. This also means that the non-ABI-stable builds test the same codepaths.